docs/vc-http-api.yml

```diff
@@ -295,7 +301,8 @@ components:
           description: The JSON-LD type of the presentation.
         "holder":
           type: object
-          description: The holder
+          description: The holder - will be ignored if no proof is present since there is no proof of authority over the credentials
+          nullable: true
         "verifiableCredential":
           type: array
```

Comment:
hmm, why is this an array? Why not just take in a single VP? -1 to unbounded arrays as inputs and outputs.

Reply:
good question, I matched what was already in the code there and did not modify it.  @OR13 any comment as to why that was originally defined as an array? personally I am fine either way as long as it is clear that an implementation may choose to only take a single element

Reply:
@msporny happy to change this to a single item, however, the issue then is that it will not be backwards compatible anymore - should we hold this until a solid refactor pass after plugfest?

Reply:
I'ts fine as is, no need to change it.